test/WebSites/RazorWebSite/project.json

```diff
@@ -23,13 +23,16 @@
         ]
       }
     },
-    "dnxcore50": {
+    "netstandardapp1.5": {
       "compilationOptions": {
         "define": [
           "DNXCORE50_CUSTOM_DEFINE"
```

Comment:
If `DNXCORE50_CUSTOM_DEFINE` became `NETSTANDARDAPP1_5_CUSTOM_DEFINE` (in code), shouldn't this match?